codestaintin/More_Recipes
# Feedback Implementation

#### What does this PR do?
- Add MIT License to project
- Add jest, enzyme, react-paginate, expect, and moxios packages to package.json
- Write client tests for actions and reducers
- Refactor project files to make codebase cleaner
- Update README.md

#### Description of Task to be completed?

- Refactor code base
- Write test suites for actions and reducers
- Add License to README.md
- Implement pagination on home page, user recipes page and search page
- Fix home page bug links

#### How should this be manually tested?
- Run ```git clone``` the repository
- Run ``` cd more-recipes```
- Run ```npm install``` to install project dependencies
- Run ```git checkout chore/feedback-implementation```
- Run ```npm start:dev``` and ```npm run build:dev``` in the terminal to launch the application
- Navigate to ```http://localhost:8080``` to see live app

#### Any background context you want to provide?
- Setup for client-side testing
- Add MIT License to project

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshot
N/A

#### Questions
N/A
